src/components/manager/runs/RunLastContent.js

```diff
@@ -0,0 +1,102 @@
+import React, {useState, useEffect} from "react";
+import {Table} from "antd";
+import {connect} from "react-redux";
+import PropTypes from "prop-types";
+import {fetchRuns} from "../../../modules/wes/actions";
+
+const LastIngestionTable = ({runs, fetchRuns}) => {
+    const [ingestions, setIngestions] = useState([]);
+
+    const processIngestions = (data) => {
+        const latestIngestions = data.reduce((acc, obj) => {
+            if (obj.state === "COMPLETE") {
+                const dataType = obj.details.request.tags.workflow_metadata.data_type;
+                const tableId = obj.details.request.tags.table_id;
```

Comment:
some runs (analysis runs) will not have table Ids - if this is undefined, you should skip this run